backends/ebpf/codeGen.cpp

```diff
@@ -429,4 +444,28 @@ void CodeGenInspector::widthCheck(const IR::Node *node) const {
             node, tb->size);
 }
 
+unsigned EBPFInitializerUtils::ebpfTypeWidth(P4::TypeMap *typeMap, const IR::Expression *expr) {
+    auto type = typeMap->getType(expr);
+    if (type == nullptr) type = expr->type;
+    if (type->is<IR::Type_InfInt>()) return 32;  // let's assume 32 bit for int type
```

Comment:
the only place where this can happen as far I know is in rhs of shifts.

Reply:
I can't recall where exactly hit the problem with this type.
For now I am able to make this statement true with slice, like `field[127:64]` (which is translated using bit shifts).